.github/workflows/chromatic.yml

```
@@ -0,0 +1,31 @@
+# .github/workflows/chromatic.yml
+
+# Workflow name
+name: "Chromatic"
+
+# Event for the workflow
+on: push
+
+# List of jobs
+jobs:
```

Can we remove the comments?

```suggestion
name: "Chromatic"

on: push

jobs:
```